src/main/java/com/spotify/reaper/service/SegmentRunner.java

```diff
@@ -318,6 +333,21 @@ boolean canRepair(RepairSegment segment, String keyspace, JmxProxy coordinator,
     return true;
   }
 
+  private boolean IsRepairRunningOnOneNode(RepairSegment segment) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)